templates/typescript_gapic/test/gapic-$service-$version.ts.njk

```diff
@@ -200,7 +226,15 @@ describe('{{ service.name }}Client', () => {
                 projectId: 'bogus', 
             });
             // Mock request
-            const request = {};
+            const request: protosTypes{{ method.inputInterface }} = {};
```

Comment:
I'm thinking that if this same sequence is repeated four times it might make sense to make it a macro (accepting `method` as a parameter), if it's possible.

Reply:
sure! put `initRequestWithHeaderParam` method in macro of `_util.njk` and import it